### PR TITLE
the Syndicate expansion part one (allegedly)

### PR DESCRIPTION
**Pull request by Wirpy**
### From the depths of the research labs over at _FleckTex Dynamics_ comes armour anew! Pre-order now at a discounted price for only 4999,99 TerraGov credits! _Limited time offer while stock lasts!_
<details>
  <summary>Current Sprites</summary>
  
the boys
the boys 2
the boys 3

</details>

<details>
  <summary>Old WIP Sprites</summary>
  
b45375dfb3036f9ca4cf80fe43fa8437

[
image
](url)

syndie_front
syndie_siide
syndie_back
</details>

### Hardsuits

Corporate liasions, marines, deltards, and to others whom it may concern we here at Flecktex Dynamics have with our great relationship with the absolutely delightful staff over at TerraGov managed to come together and make a great deal sending our patented hardsuit designs straight to the frontlines where Terrragov needs em the most!

Introducing the Mark I, III and V hardsuits ready for assembly and use avaliable in a armour vendor near you marine! Designed to withstand even the mightiest blows from any threat! Durable, yet economic and compatiable with jaeger modules. What more does one want or need? We'll make sure your organs stays inside! 
According to our tests our suits will increase survability against any xenomorph threat by 16.33%! 

If maints wants me to del the hardsuit tab in vendor and put em in style line thats ok but I thought It'd be more clear on what works with what by putting em in their own tab. 

## TL:DR
Three sets of Syndicate inspired armour including webbing variants (They're all modular lego like mk1 jaeger)
Hyperscale colour palettes (including visors!) 

All sprites beyond the light helmet (mk I), exoskeleton and webbing are approved + the colours. (waiting for muh kuro approval for those) 

## Usage Guide
The Exoskeleton for the hardsuits is hyperscaled and uses the same palette as the armour. 

Overall it works pretty much the same as Jaeger Mk1.

Step 1; Vend exoskeleton and desired armour pieces.

Step 2; Use facepaint to colour exoskeleton and armour pieces into desired colours

Step 2.1; By right clicking with face paint there is more functionality like a secondary colour layer and currently a webbing variant that can be applied on the chests (Which is also hyperscaled and colorable thru right click)

Currently it's only the helmets and legs that have a secondary colour layer which you change change around and the webbing variant on the chest pieces. 

3. Assemble the armour by adding the pieces onto the exoskeleton

4. Wear and enjoy 

## Why It's Good For The Game

More customization and drip for marines, cool content drop. I LOVE HARDSUITS AND THE SYNDICATE!!!

Three brand new hardsuits (May contain previous owner), _YOU HEARD IT RIGHT!_ **_HARDSUITS BABY!_**
- Light (Mark I)
- Medium (Mark III)
- Heavy (Mark V) 

New visor colours
- Syndicate Black
- Syndicate Red
- Dark Blue
- Space Cadet 
- Nightshade

All the armour pieces are lego, they are using their own exoskeleton that works pretty much the same way Jaeger Mk1 does. 
They are not compatible with mk1 armour pieces due them not mixing well together but all jaeger modules are compatible. 

Syndicate colours have also been adjusted to the full word syndicate instead of syndie red etc 

The future, the lore and the what?
- Brings the Syndicate towards reality with there being factual proof of em existing!
- Potential usage for a Syndicate faction in the campaign HvH mode?
- Potential usage for a hostile Syndicate ERT? 
- More content to come allegedly 
- I have begun writing the lore

Huge thanks to both Hyper and Artur whom have helped me a lot with this PR and also the fine folks in armored kurtwork such as Kuro, Orca and Gregor. 

## Changelog
:cl: wirpy, artur
add: Gorlex Red (Visor+Armor), Syndicate Red (Visor), Syndicate Black (Visor), Nightshade (Visor), Syndicate Green (Visor), Dark Blue (Visor), Space Cadet (Visor) 
add: Mark V Breacher Armour (Heavy)
add: Mark III Marauder Armour (Medium)
add: Mark I Raider Armour (Light)
add: Webbing variant chest pieces
add: Hardsuit Exoskeleton (Hyperscaled)
add: Variant functionality for Armour (You can toggle a webbing on the chest pieces)
add: Secondary hyperscale colour layer for armour pieces (Currently only on legs and helmet) 
qol: Hardsuit vending tab 
/:cl:

### CODE REVIEW

**Comment by Pariah919** [comment]
Not these goobers

**Comment by yyzsong** [comment]
ass armour

**Comment by nikothedude** [comment]
cheeked up

**Comment by Sirsnek15** [comment]
assless chaps

**Comment by zzurge** [comment]
WHY THE SYNDIE CAKED UP LIKE THAT FOR

**Comment by Tommy-211** [comment]
when

**Review by Kuro020 — APPROVED** [review]

**Comment by lKiseki** [comment]
Looks good to merge otherwise.

File: code/__DEFINES/greyscale_guns.dm [hunk 1]
```diff
@@ -19,6 +19,12 @@
 #define GUN_PALETTE_PURPLE "#222046#36386b#535185#726dc2"
 #define GUN_PALETTE_AQUA "#264188#28599a#1c96c4#36f5e9"
 #define GUN_PALETTE_DARK_GREEN "#1a301a#264227#385d39#508151"
+#define GUN_PALETTE__SYNDICATE_RED_GUN "#3f0f25#540f23#b72b2b#ea4a3d"
+#define GUN_PALETTE__GORLEX_RED_GUN "#29272b#4b2e44#9f3139#b54711"
+#define GUN_PALETTE__DARK_BLUE_GUN "#151126#202045#283d7d#275591"
+#define GUN_PALETTE__SPACE_CADET_GUN "#151126#202946#224577#215ead"
+#define GUN_PALETTE__SYNDICATE_BLACK_GUN "#181719#232226#545350#63615e"
+#define GUN_PALETTE__NIGHTSHADE_GUN "#080808#141416#25252b#2d2d37"
```
[the remark this answers is not in the document]
Comment (Wirpy) [reply code/__DEFINES/greyscale_guns.dm:27]: balded ig

File: code/__DEFINES/greyscale_guns.dm [hunk 2]
```diff
@@ -36,4 +42,10 @@
 	"Gun Metal Blue" = GUN_PALETTE_BLUE, \
 	"Gold" = GUN_PALETTE_GOLD, \
 	"Aqua" = GUN_PALETTE_AQUA, \
+	"Syndicate Red" = GUN_PALETTE__SYNDICATE_RED_GUN, \
+	"Syndicate Black" = GUN_PALETTE__SYNDICATE_BLACK_GUN, \
+	"Gorlex Red" = GUN_PALETTE__GORLEX_RED_GUN, \
+	"Dark Blue" = GUN_PALETTE__DARK_BLUE_GUN, \
+	"Space Cadet" = GUN_PALETTE__SPACE_CADET_GUN, \
+	"Nightshade" = GUN_PALETTE__NIGHTSHADE_GUN, \
```
[the remark this answers is not in the document]
Comment (Wirpy) [reply code/__DEFINES/greyscale_guns.dm:50]: prob balded here too

**Comment by 1kajowsmol** [comment]
need merge......

**Comment by BoltedAirlock** [comment]
syndiebros... it's done

**Pull request merged**